rawspec.c

```diff
@@ -420,9 +446,11 @@ char tmp[16];
   // Init user_data to be array of callback data structures
   ctx.user_data = &cb_data;
 
-  // Zero-out the callback data sructures
+  // Zero-out the callback data sructures.
+  // Turn on dynamic debugging if requested.
   for(i=0; i<ctx.No; i++) {
     memset(&cb_data[i], 0, sizeof(callback_data_t));
+    cb_data[i].debug_callback = flag_debugging;
```

Comment:
Perhaps `= flag_debugging | DEBUG_CALLBACKS` is better than having a disparately placed L762

Reply:
I would actually like to get rid of DEBUG_CALLBACKS

Reply:
Yeah maybe now is a good time to do so.. Seems like overkill to have 2 mechanisms. :+1: